cmd/main.go

```diff
@@ -81,6 +81,7 @@ func main() {
 			profileFlags(),
 			statusFlags(),
 			warmupFlags(),
+			backupFlags(),
```

Comment:
We could have dump/load as separate command

Reply:
c139306